src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -699,6 +701,7 @@ bool Dataloader::_load_map_dir(GameManager& game_manager) const {
 	F(continent) \
 	F(adjacencies) \
 	F(region) \
+	F(region_colours) \
```

Comment:
Because the region colours file isn't customised, having it here in the list of things which we expect in `default.map` causes an error message. This line just needs to be removed, the `region_colours` variable created above will provide the path.